### PR TITLE
General Cleanup 6

### PR DESCRIPTION
**Pull request by hensldm**
This cleanup PR focuses on removing the leading 0's from the offset comments in actors.

### CODE REVIEW

File: src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.h [hunk 1]
```diff
@@ -8,9 +8,9 @@ struct BgDyYoseizo;
 typedef void (*BgDyYoseizoActionFunc)(struct BgDyYoseizo*, PlayState*);
 
 typedef struct BgDyYoseizo {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ BgDyYoseizoActionFunc actionFunc;
-    /* 0x0148 */ char unk_148[0x33BC];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ BgDyYoseizoActionFunc actionFunc;
+    /* 0x148 */ char unk_148[0x33BC];
 } BgDyYoseizo; // size = 0x3504
```
Comment (engineer124) [inline src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.h]: This should stay 4 digits as the size is also 4 digits

File: src/overlays/actors/ovl_Demo_Shd/z_demo_shd.h [hunk 2]
```diff
@@ -6,8 +6,8 @@
 struct DemoShd;
 
 typedef struct DemoShd {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_0144[0x118C];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ char unk_0144[0x118C];
 } DemoShd; // size = 0x12D0
```
Comment (engineer124) [inline src/overlays/actors/ovl_Demo_Shd/z_demo_shd.h:11]: Same here

File: src/overlays/actors/ovl_En_Encount2/z_en_encount2.h [hunk 3]
```diff
@@ -21,16 +21,16 @@ typedef struct EnEncount2Effect{
...
+    /* 0x15C */ EnEncount2ActionFunc actionFunc;
+    /* 0x160 */ s16 deathTimer;
+    /* 0x162 */ s16 isPopped;
+    /* 0x164 */ s16 switchFlag;
+    /* 0x168 */ f32 scale;
+    /* 0x16C */ f32 oscillationAngle;
+    /* 0x170 */ ColliderJntSph collider;
+    /* 0x190 */ ColliderJntSphElement colElement;
+    /* 0x1D0 */ EnEncount2Effect effects[EN_ENCOUNT2_EFFECT_COUNT];
 } EnEncount2; // size = 0x2A70
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Encount2/z_en_encount2.h:34]: Here too

File: src/overlays/actors/ovl_En_Jso2/z_en_jso2.h [hunk 4]
```diff
@@ -8,10 +8,10 @@ struct EnJso2;
...
 typedef struct EnJso2 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x134];
-    /* 0x0278 */ EnJso2ActionFunc actionFunc;
-    /* 0x027C */ char unk_27C[0xE14];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ char unk_144[0x134];
+    /* 0x278 */ EnJso2ActionFunc actionFunc;
+    /* 0x27C */ char unk_27C[0xE14];
 } EnJso2; // size = 0x1090
```
Comment (engineer124) [inline src/overlays/actors/ovl_En_Jso2/z_en_jso2.h:15]: Here too

File: src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.h (hunk 1 above)
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.h:14]: This is definitely going to need 4 digits.

File: src/overlays/actors/ovl_En_GirlA/z_en_girla.h [hunk 5]
```diff
@@ -127,8 +127,8 @@ typedef struct {
 } StickDirectionPrompt; // size = 0x38
 
 typedef struct ShopItem {
-    /* 0x00 */ s16 shopItemId;
-    /* 0x02 */ Vec3s spawnPos;
+    /* 0x0 */ s16 shopItemId;
+    /* 0x2 */ Vec3s spawnPos;
 } ShopItem; // size = 0x8;
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_GirlA/z_en_girla.h:132]:
```suggestion
} ShopItem; // size = 0x8
```

File: src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h [hunk 6]
```diff
@@ -10,39 +10,39 @@ typedef void (*EnMushi2ActionFunc)(struct EnMushi2*, PlayState*);
 #define ENMUSHI2_GET_3(thisx) ((thisx)->params & 3)
 
 typedef struct {
-    /* 0x00 */ s16 unk_00;
-    /* 0x02 */ s16 unk_02;
+    /* 0x0 */ s16 unk_00;
+    /* 0x2 */ s16 unk_02;
 } EnMushi2Struct;
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h:15]: Should remove the `0x`s to be consistent, and add the size

File: src/overlays/actors/ovl_En_Water_Effect/z_en_water_effect.h [hunk 7]
```diff
@@ -19,11 +19,11 @@ typedef struct {
...
-    /* 0x309 */ ENWATEREFFECT_777 = 777,
-    /* 0x30A */ ENWATEREFFECT_778,
-    /* 0x30B */ ENWATEREFFECT_779,
-    /* 0x30C */ ENWATEREFFECT_780,
+    /* 001 */ ENWATEREFFECT_1 = 1,
+    /* 777 */ ENWATEREFFECT_777 = 777,
+    /* 778 */ ENWATEREFFECT_778,
+    /* 779 */ ENWATEREFFECT_779,
+    /* 780 */ ENWATEREFFECT_780,
 };
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Water_Effect/z_en_water_effect.h:27]: The point of using hex on enums is to make it easy to see in memory or in the code, don't think this is an appropriate change; if anything the numbers should change to reflect the hex values.

File: src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.h [hunk 8]
```diff
@@ -40,23 +40,23 @@ enum {
 };
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.h:40]: Same here

File: src/overlays/actors/ovl_Mir_Ray/z_mir_ray.h [hunk 9]
```diff
@@ -9,35 +9,35 @@ struct MirRay;
...
+    /* 01 */ MIRRAY_SPIRIT_SUNBLOCKROOM_DOWNLIGHT,
+    /* 02 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_DOWNLIGHT,
+    /* 03 */ MIRRAY_SPIRIT_ARMOSROOM_DOWNLIGHT,
+    /* 04 */ MIRRAY_SPIRIT_TOPROOM_DOWNLIGHT,
+    /* 05 */ MIRRAY_SPIRIT_TOPROOM_CEILINGMIRROR,
+    /* 06 */ MIRRAY_SPIRIT_SINGLECOBRAROOM_COBRA,
+    /* 07 */ MIRRAY_SPIRIT_TOPROOM_COBRA1,
+    /* 08 */ MIRRAY_SPIRIT_TOPROOM_COBRA2,
+    /* 09 */ MIRRAY_GANONSCASTLE_SPIRITTRIAL_DOWNLIGHT,
     /* 10 */ MIRRAY_MAX,
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Mir_Ray/z_mir_ray.h:22]: Can't do that, 0-prefixed numbers in C are octal (yes, stupid, but true). Use spaces instead.

File: src/overlays/actors/ovl_Obj_Boyo/z_obj_boyo.h [hunk 10]
```diff
@@ -10,7 +10,7 @@ typedef void (*BumperCollideActorFunc)(struct ObjBoyo*, void*);
 typedef struct {
     /* 0x0 */ s16 id;
     /* 0x4 */ BumperCollideActorFunc actorCollideFunc;
-} BumperCollideInfo;
+} BumperCollideInfo; // size = 0x8
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Boyo/z_obj_boyo.h:13]: Need to make these consistent one way or another.

File: src/overlays/actors/ovl_Obj_Iceblock/z_obj_iceblock.h [hunk 11]
```diff
@@ -27,8 +27,8 @@ typedef struct {
 } ObjIceBlockUnkStruct2; // size = 0x10
 
 typedef struct {
-    f32 unk_00;
-    f32 unk_04;
+    /* 0x0 */ f32 unk_00;
+    /* 0x4 */ f32 unk_04;
 } ObjIceBlockUnkStruct3; // size = 0x8
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Iceblock/z_obj_iceblock.h:32]: And here (Vec2f?)

File: src/overlays/actors/ovl_Obj_Skateblock/z_obj_skateblock.h [hunk 12]
```diff
@@ -10,26 +10,26 @@ typedef void (*ObjSkateblockActionFunc)(struct ObjSkateblock*, PlayState*);
 #define OBJSKAEBLOCK_GET_F(thisx) ((thisx)->params & 0xF)
 
 typedef struct {
-    /* 0x00 */ CollisionPoly* poly;
-    /* 0x04 */ f32 height;
-    /* 0x08 */ s32 bgId;
+    /* 0x0 */ CollisionPoly* poly;
+    /* 0x4 */ f32 height;
+    /* 0x8 */ s32 bgId;
 } ObjSkateblockFloor; // size = 0xC
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Skateblock/z_obj_skateblock.h:16]: And here

File: src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h [hunk 13]
```diff
@@ -34,26 +34,26 @@ typedef struct {
 } ObjSpinyrollStruct2; // size = 0xF8?
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h:34]: ?

File: src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h (hunk 6 above)
Comment (hensldm) [reply src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h:15]: Usage does not suggest a normal `Vec2s`. Also I think I prefer `0x` for these, I'll change to this style if that is okay.

Comment (hensldm) [reply src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.h:15]: As far as I know all structs should have a `0x`, it is the enums that are inconsistent with each other and with structs.

File: src/overlays/actors/ovl_Obj_Iceblock/z_obj_iceblock.h (hunk 11 above)
Comment (hensldm) [reply src/overlays/actors/ovl_Obj_Iceblock/z_obj_iceblock.h:32]: Not quite sure, probably best to wait for documentation.

File: src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h (hunk 13 above)
Comment (hensldm) [reply src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h:34]: ?? :) (see the alternative testing piece, still need to figure what these structs actually are)

**Review by EllipticEllipsis — APPROVED** [review]

**Review by jpburnett — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**